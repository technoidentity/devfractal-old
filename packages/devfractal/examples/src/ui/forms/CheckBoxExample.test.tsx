import React from 'react'

import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { CheckBox } from './CheckBoxExample'

test('CheckBox Example', () => {
  const component: ReactTestRenderer = renderer.create(<CheckBox />)
  expect(component.toJSON()).toMatchSnapshot()
})
