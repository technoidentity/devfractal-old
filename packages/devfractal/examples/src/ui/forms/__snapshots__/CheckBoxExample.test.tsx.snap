// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckBox Example 1`] = `
Array [
  <section
    className="section"
  >
    <h1
      className="title is-4"
    >
      Dynamic Checkbox
    </h1>
    <div
      className="control"
    >
      <label
        className="checkbox"
      >
        <input
          checked={false}
          className="checkbox"
          name="checkbox"
          onChange={[Function]}
          type="checkbox"
        />
         
        UnChecked
      </label>
    </div>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title is-4"
    >
      Checkbox
    </h1>
    <div
      className="control"
    >
      <label
        className="checkbox"
      >
        <input
          checked={true}
          className="checkbox"
          readOnly={true}
          type="checkbox"
        />
         
        Remember me
      </label>
    </div>
  </section>,
  <section
    className="section"
  >
    <h1
      className="title is-4"
    >
      Add links 
    </h1>
    <div
      className="control"
    >
      <label
        className="checkbox"
      >
        <input
          className="checkbox"
          onChange={[Function]}
          type="checkbox"
        />
         
        I agree 
        <a
          href="#!"
        >
          terms and conditions
        </a>
      </label>
    </div>
  </section>,
  <section
    className="section"
  >
    <div
      className="field"
    >
      <h1
        className="title is-4"
      >
        Disabled 
      </h1>
      <div
        className="control"
      >
        <label
          className="checkbox"
        >
          <input
            className="checkbox"
            disabled={true}
            type="checkbox"
          />
           Save my Changes
        </label>
      </div>
    </div>
  </section>,
]
`;
